spring-cloud-azure-feature-management/README.md

```diff
@@ -151,3 +151,113 @@ Some feature filters require parameters to decide whether a feature should be tu
+              time-window-filter-setting-end: "Mon, 01 July 2019 00:00:00 GMT"
+```
+
+### TargetingFilter
+
+This filter provides the capability to enable a feature for a target audience. An in-depth explanation of targeting is explained in the targeting section below. The filter parameters include an audience object which describes users, groups, and a default percentage of the user base that should have access to the feature. Each group object that is listed in the target audience must also specify what percentage of the group's members should have access. If a user is specified in the users section directly, or if the user is in the included percentage of any of the group rollouts, or if the user falls into the default rollout percentage then that user will have the feature enabled.
+
+```yml
+feature-management:
+  target:
+    enabled-for:
+      - 
+        name: targetingFilter
+        parameters:
+          users:
+            - Jeff
+            - Alicia
+          groups:
+            - 
+              name: Ring0
+              rolloutPercentage: 100
+            - 
+              name: Ring1
+              rolloutPercentage: 100
+          defaultRolloutPercentage: 50
+```
+
+## Targeting
+
+Targeting is a feature management strategy that enables developers to progressively roll out new features to their user base. The strategy is built on the concept of targeting a set of users known as the target audience. An audience is made up of specific users, groups, and a designated percentage of the entire user base. The groups that are included in the audience can be broken down further into percentages of their total members.
+
+The following steps demonstrate an example of a progressive rollout for a new 'Beta' feature:
+
+Individual users Jeff and Alicia are granted access to the Beta
+Another user, Mark, asks to opt-in and is included.
+Twenty percent of a group known as "Ring1" users are included in the Beta.
+The number of "Ring1" users included in the beta is bumped up to 100 percent.
+Five percent of the user base is included in the beta.
+The rollout percentage is bumped up to 100 percent and the feature is completely rolled out.
+This strategy for rolling out a feature is built in to the library through the included TargetingFilter feature filter.
+
+### Targeting in an Application
+
+An example web application that uses the targeting feature filter is available in the [FeatureFlagDemo](https://github.com/microsoft/spring-cloud-azure/tree/master/spring-cloud-azure-samples/feature-management-web-sample) example project.
+
+To begin using the `TargetingFilter` in an application it must be added as a `@Bean` like any other Feature Filter. Unlike the other Filters the `TargetingFilter` relies on another `@Bean` to be added to the application, `ITargetingContextAccessor`. The `ITargetingContextAccessor` allows for defining the current `TargetingContext` to be used for defining the current users; id, and groups. An example of this is:
```

Comment:
Let's add a link to your implementation of TargetingContextAccessor in the example that pulls data from the requestContext.